dataland-internal-storage/src/test/resources/db/migration/V6/expectedTransformedDatabaseEntry.json

```diff
@@ -0,0 +1,121 @@
+{
+  "general": {
+    "fiscalYearDeviation": "Deviation",
+    "fiscalYearEnd": "2022-10-05",
+    "nfrdMandatory": null,
+    "numberOfEmployees": null,
+    "referencedReports": {
+      "SustainabilityReport": {
+        "reference": "50a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+        "reportDate": null,
+        "isGroupLevel": null,
+        "currency": "THB"
+      },
+      "ESEFReport": {
+        "reference": "50a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+        "reportDate": null,
+        "isGroupLevel": null,
+        "currency": "BSD"
+      }
+    },
+    "assurance": {
+      "assurance": "LimitedAssurance",
+      "provider": null,
+      "dataSource": {
+        "page": 91,
+        "report": "SustainabilityReport",
+        "tagName": "initiatives"
+      }
+    },
+    "scopeOfEntities": "NA",
+    "euTaxonomyActivityLevelReporting": "No",
+    "alignedActivities": [
+      {
+        "substantialContributionCriteria": {
```

Comment:
This seems to be incorrect. This is the old structure and the old naming. (Compare to lines 55 and onwards)

Reply:
Has been discussed

Reply:
Deleted, ok